src/components/About/About.tsx

```diff
@@ -0,0 +1,51 @@
+import { Anchor, Container, Grid, Group, Image, Modal, Text, Title } from '@mantine/core';
+
+import { useAppSelector } from '@/redux/hooks';
+
+interface Props {
+  opened: boolean;
+  close: () => void;
+}
+export function About({ opened, close }: Props) {
+  const openSpaceVersion = useAppSelector((state) => state.version.openSpaceVersion);
+
+  function osVersionNumber(): string {
+    if (!openSpaceVersion) {
+      return 'Fetching OpenSpace version...';
+    }
+
+    function formatVersion(version: {
```

Comment:
Create a type for the version type in types.d.ts? That would make this a a lot easier to read. E.g.

```typescript
export interface OpenSpaceVersionNumber { 
  major: number;
  minor: number;
  patch: number;
}
```